src/components/homebar.js

```diff
@@ -25,7 +25,7 @@ const Homebar = () => (
             <Link to="/favorites"><NavImg src={favorites} /></Link>
         </Nav.Item>
         <Nav.Item>
-            <Link to="/"><NavImg src={home} /></Link>
+            <Link to="/home"><NavImg src={home} /></Link>
```

Comment:
Check week-7 code again, this should link to "/"

Reply:
fixed